src/data/general-cs-quiz.ts

```diff
@@ -1565,6 +1565,38 @@ const generalCSQuiz = [
     Explanation:
       "A Linked List contains nodes and each node has data and a reference to next node so that it can have access to the data of its next node and we can get the information of all nodes by iterating the node until it reaches nullptr.",
     Link: "https://www.freecodecamp.org/news/how-linked-lists-work/"
+  },
+  {
+    Question: "What is the Basis for a Red-Black tree?",
```

Comment:
The letter B doesn't need to be capitalized here

```suggestion
    Question: "What is the basis for a Red-Black tree?",
```